test/redbird/crypto_test.exs

```diff
@@ -0,0 +1,96 @@
+defmodule Redbird.CryptoTest do
+  use Redbird.ConnCase, async: true
+
+  alias Redbird.Crypto
+
+  describe "sign_key/3" do
+    setup do
+      on_exit(fn -> Application.delete_env(:redbird, :key_base) end)
+    end
+
+    test "signs the key with configured key base" do
+      Application.put_env(:redbird, :key_base, "abcdef12345")
```

Comment:
Do you know if we need to set the test to `async: false` since we put this environment variable? I know we delete it `on_exit/2`, but I still think that this test could put this application ENV var and another test read it if they're running concurrently?

Reply:
Yes, great catch. I had intended to flip that flag.